### PR TITLE
chore: merge database gen scripts

### PR DESCRIPTION
**Pull request by kylecarbs**

### CODE REVIEW

File: coderd/database/db.go [hunk 1]
```diff
@@ -18,6 +18,13 @@ import (
 	"golang.org/x/xerrors"
 )
 
+// Well-known lock IDs for lock functions in the database. These should not
+// change. If locks are deprecated, they should be kept to avoid reusing the
+// same ID.
+const (
+	LockIDDeploymentSetup = iota + 1
```
Comment (ammario) [inline coderd/database/db.go:25]: Why `iota + 1`?

**Review by ammario — APPROVED** [review]
Solid reduction in LoC

I didn't give this a thorough review since it's all dev code.

File: coderd/database/db.go (hunk 1 above)
Comment (kylecarbs) [reply coderd/database/db.go:25]: This is just how it was... seemed risky to change as part of this.

Comment (mafredri) [reply coderd/database/db.go:25]: I don't know what the original reason was, but a typical reason is to avoid mixing it with the zero value. FWIW.

**Pull request merged**